.gitignore

```diff
@@ -0,0 +1,154 @@
+### Python template
```

Comment:
Please don't check this in. `cp .gitignore.template .gitignore` and modify as you like.